com.unity.render-pipelines.core/Editor/Lighting/ProbeVolume/ProbeVolumeUI.Drawer.cs

```diff
@@ -111,7 +111,7 @@ static void Drawer_ToolBar(SerializedProbeVolume serialized, Editor owner)
 
         static void Drawer_VolumeContent(SerializedProbeVolume serialized, Editor owner)
         {
-            if (!ProbeReferenceVolume.instance.isInitialized)
+            if (!ProbeReferenceVolume.instance.isInitialized || !ProbeReferenceVolume.instance.enabledBySRP)
```

Comment:
When can this happen?
Start with APV enabled in the asset then switch it off?
Sounds like we should cleanup the APV in this case no?

Reply:
It can be valid but not enabled by current frame settings

Reply:
Ah right, forgot about the frame settings and thought it was only the global settings.